desktop/apps/categories3/components/Gene.js

```diff
@@ -1,16 +1,31 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import styled from 'styled-components'
 
 const propTypes = {
   id: PropTypes.string.isRequired,
   name: PropTypes.string.isRequired
 }
 
+const GeneItem = styled.li`
+  font-size: 20px;
+  line-height: 1.33em;
+  margin-bottom: 0.66em;
+`
+const GeneLink = styled.a`
+  text-decoration: none;
+
+  &:hover {
+    color: purple;
```

Comment:
Should we be pulling in the artsy-purple color here?

Reply:
Good catch, will do a pass to check over any colors being specified in css. Since this is outside of our Stylus workflow, I should just import that purple directly from `stylus_lib/colors.json`, I presume?